src/presentation/pages/Remarkables/Main/Main.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Link, Typography } from '@material-ui/core';
+import useTranslation from 'app/hooks/useTranslation';
+import styles from './Main.module.scss';
+import { HowItWorks } from '../HowItWorks/HowItWorks';
+import { BackgroundImageLayout } from 'infrastructure/components/BackgroundImageLayout/BackgroundImageLayout';
+import { AthElitesCarousel } from 'presentation/components/AthElitesCarousel/AthElitesCarousel';
+import { carouselItems } from 'app/constants/athletes/remarkablesCarousel';
+
+export const Main = () => {
+  const t = useTranslation();
+
+  return (
+    <>
+      <BackgroundImageLayout>
+        <div className={styles.mainContent__header}>
```

Comment:
What do you think about creating a component for the children?

Reply:
Discussed in DM. Change will be added in a new PR.